DataFormats/L1TCalorimeterPhase2/src/DigitizedClusterGT.cc

```diff
@@ -0,0 +1,8 @@
+#include <bitset>
+
+#include "DataFormats/L1TCalorimeterPhase2/interface/DigitizedClusterGT.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+void l1tp2::DigitizedClusterGT::print(const std::string location) {
+  std::cout << "[Info:] DigitizedClusterGT " << location << ": " << std::bitset<64>{data()} << std::endl;
+}
```

Comment:
This can't be printed.

Reply:
Removed method as of [e7e753d](https://github.com/cms-l1t-offline/cmssw/pull/1069/commits/e7e753df69e7aff845a3b3dfbaed9a06c9fdaec6)